### ISSUE TITLE
Question about noise removal

### ISSUE THREAD

**Issue by bethac07** [cytomining/pycytominer#242]
Purely a question - was it ever considered to use CV rather than pure standard deviation in this feature? This is an obviously artificial example, but to me I would feel 'b' is more noisy in that across my feature the fold change is really large, but since 'a' has a higher absolute value it gets removed first

<img width="1016" alt="image" src="https://user-images.githubusercontent.com/6721515/202228498-67087ae9-ed3d-4b85-bd16-20a416ef6d6d.png">

**Comment by gwaybio** [comment]
> was it ever considered to use CV rather than pure standard deviation in this feature?

I don't believe so. @ruifanp developed the approach, so he might have some more insight.

In general, I think that we're definitely open to adding functionality here!
